cypress/support/step_definition/course_board/createEditDeleteMultiColumnInBoardSteps.spec.js

```diff
@@ -16,7 +16,7 @@ When("I click on the option Edit in three dot menu in course board", () => {
 	board.clickOnEditInThreeDotCourseBoardTitle();
 });
 
-Then("I enter the course board title {string}", (boardTitle) => {
+Then("I enter the room board title {string}", (boardTitle) => {
```

Comment:
why was it changed? the position of this file as well as the usage of it speaks as if it would be used for the old courses and not the new room page, so why was it changed?

Reply:
yes it has be changed to the 'course', this is for the old course board.